package.json

```diff
@@ -21,7 +21,7 @@
     "yargs": "^12.0.2"
   },
   "engine": {
-    "node": ">=6.0"
+    "node": ">=8.0"
```

Comment:
bumped engine to 8 because travis ci is against 8 - let me know if 6 needs to be supported (if so I'll need to adjust syntax)

Reply:
no need to support 6